### ISSUE TITLE
Martin needs to watch Trainspotting

### ISSUE THREAD

**Issue by terrajobst** [martinwoodward/martin_public#2]
**Version Used**: 

Martin Woodward, Jan 22 2017

**Steps to Reproduce**:

- [x] Find Martin
- [ ] Talk about movies
- [ ] Mention the worst toilet in Scotland

**Expected Behavior**:

"Yeah, that was a bloody good moment in Trainspotting!"

**Actual Behavior**:

*blankstares*

### Update 1

I can still repro this in March 2019.

### Update 2

I can still repro this in July 2020. 

![](https://media.giphy.com/media/NJkIw5wfnM3e0/giphy.gif)

**Comment by anaisbetts** [comment]
It is v good, :+1:

**Comment by martinwoodward** [comment]
Thanks for the feedback. I am able to reproduce the issue - added to the backlog.

**Comment by terrajobst** [comment]
It has been four months now.

MARTIN DOES NOT CARE ABOUT THE COMMUNITY. LIFE IS MEANINGLESS. I'LL SWITCH TO BRAINFUCK.

**Comment by martinwoodward** [comment]
Yeah, sorry about that. Still not moved anywhere near the top of the backlog to be implemented.  Current backlog:

1. Sully
2. Guardians of the Galaxy Vol 2
3. Finish The Night Manager TV series

Thanks for your feedback.

**Comment by terrajobst** [comment]
Is there a Slack channel? It seems there is a lot of backroom decision making going on that doesn't involve us. Sad!

**Comment by nulltoken** [comment]
![image](http://www.reactiongifs.com/wp-content/gallery/popcorn-gifs/yNCVL.gif)

**Comment by ethomson** [comment]
Ugh, such slow adoption...!  In the meantime, upstream has moved on to Trainspotting 2.0.

**Comment by nulltoken** [comment]
I wonder... Maybe could we help Martin and create issues with the detailed synopses for the movies listed above in his backlog.

This way he could skip the (boring) watching part and catch up more easily.

Thoughts?

**Comment by martinwoodward** [comment]
Thing is, I already know the plots of 1 & 3 and I doubt the plot of 2 is particularly important otherwise it's not the movie I was hoping to see.

**Comment by nulltoken** [comment]
![](https://images.gr-assets.com/hostedimages/1381027692ra/3885099.gif)

**Comment by terrajobst** [comment]
Any updates? It's now June!

**Comment by Drawaes** [comment]
Watch 4 Rooms...

**Comment by ObsidianPhoenix** [comment]
Is there an issue if you use the any of the following variables? I need to know before I adopt.

1. Donny Darko
2. Beetlejuice
3. Clerks
4. Office Space

**Comment by martinwoodward** [comment]
Out of those, I've only seen Beetlejuice. Refusing to see Office Space may or may not be the reason I had to stop working for @shanselman.

Though I've never seen Donnie Darko, the Gary Jules version of Mad World is one of my favorite songs, which is just odd I grant you.

**Comment by martinwoodward** [comment]
A quick update to this issue. You'll be pleased to know Guardians of the Galaxy Vol 2 is now marked as resolved. Sully has been cached to check off during a particularly long flight on Fathers day (this Sunday).

At the moment there appears to be a recurrent issue with "Sing" as it appears to keep popping back to the top of the stack rank at the primary site. Hoping that the airplane flight will resolve that issue.

**Comment by ObsidianPhoenix** [comment]
I've heard other services having issues with items such as Sing. It is usually the result of rampant child processes.

The prevailing recommendations are to avoid using `kill` to deal with these processes, as it can often lead to immediate lockdown of the parent process for extended periods of time. Instead, the suggestion is usually to push those processes onto a secondary output display.

**Comment by martinwoodward** [comment]
Just saw Baby Driver. Didn't know we still made movies like that.

**Comment by terrajobst** [comment]
Nice, so progress is being made. But it looks a bit like feature creep ;-)

**Comment by davkean** [comment]
@martinwoodward The before-death milestone is missing a due date, little hard to see when this issue will be resolved.

**Comment by terrajobst** [comment]
FIX MY BUG KNOW OR GIVE ME MY MONEY BACK

**Comment by mhmd-azeez** [comment]
@martinwoodward there are strong indications that 2020 is the last of humanity. You need to fix this bug if we're going to commit to the milestone's deadline

**Comment by ObsidianPhoenix** [comment]
🎉🎉

**Comment by terrajobst** [comment]
Can’t wait to test this next time we meet! ❤️
